### ISSUE TITLE
Allow create gluster without manage lvm volumes

### ISSUE THREAD

**Issue by theundefined** [shortdudey123/chef-gluster#76]
I would like to create glusterfs on previously created volumes. I've added empty gluster.server.disks table to not create partitions there, then I've added brick_mount_path to correct dir, set volumes... but it doesn't work. I'ave received error:

```
 NoMethodError
  -------------
  undefined method `[]' for nil:NilClass

```
on line:
`   24>>     peer_bricks = chef_node['gluster']['server']['volumes'][volume_name]['bricks'].select { |brick| brick.include? volume_name }
`

**Comment by shortdudey123** [comment]
Currently only LVM managed bricks are supported.  The brick list is created in the [`server_setup` recipe](https://github.com/shortdudey123/chef-gluster/blob/master/recipes/server_setup.rb#L50-L52).

Have you tried setting that yourself?
`node.default['gluster']['server']['volumes'][volume_name]['bricks'] = ['/mnt/data/brick1/', etc...]`

**Comment by theundefined** [comment]
it doesn't work.
I have manually configured:
volume name "staticfiles", bricks in /srv/space/staticfiles
servers : node1 and node2

"by hand" i created it using something like
```
gluster volume create staticfiles rep 2 transport tcp node{1..2}:/srv/space/staticfiles
gluster volume start staticfiles
mount -t glusterfs node1:/staticfiles /srv/staticfiles

```
(of course after probing servers)

I've tried to recreate that configuration using chef by setting attributes:

```
    "gluster": {
      "server": {
        "disks": [   ],
        "brick_mount_path": "/srv/space/",
        "volumes": {
          "staticfiles": {
            "peers": [
              "node1",
              "node2"
            ],
            "replica_count": 2,
            "volume_type": "replicated",
            "bricks": [
              "/srv/space/staticfiles/"
            ]
          }
        }

```

**Comment by shortdudey123** [comment]
I can look at adding support for existing disks.  Feel free to submit a PR if you with though :)

**Comment by theundefined** [comment]
please, https://github.com/shortdudey123/chef-gluster/pull/77 :-)

**Comment by shortdudey123** [comment]
Released under `v5.2.0`
